x-pack/plugins/remote_clusters/__jest__/client_integration/helpers/remote_clusters_actions.ts

```diff
@@ -0,0 +1,182 @@
+  };
+  setProxyAddress: (proxyAddress: string) => void;
+  serverNameInput: {
+    label: () => string;
+  };
+  saveButton: {
+    click: () => void;
+    isDisabled: () => boolean;
+  };
+  getErrorMessages: () => string[];
+  globalErrorExists: () => boolean;
+}
+export const createRemoteClustersActions = (testBed: TestBed): RemoteClustersActions => {
+  const { form, exists, find, component } = testBed;
+
+  const docsButtonExists = () => exists('remoteClusterDocsButton');
+  const createPageTitleActions = () => {
+    const pageTitleSelector = 'remoteClusterPageTitle';
+    return {
+      pageTitle: {
+        exists: () => exists(pageTitleSelector),
+        text: () => find(pageTitleSelector).text(),
+      },
+    };
+  };
+  const createNameInputActions = () => {
+    const nameInputSelector = 'remoteClusterFormNameInput';
+    return {
+      nameInput: {
+        setValue: (name: string) => form.setInputValue(nameInputSelector, name),
+        getValue: () => find(nameInputSelector).props().value,
+        isDisabled: () => find(nameInputSelector).props().disabled,
+      },
+    };
+  };
+
+  const createSkipUnavailableActions = () => {
+    const skipUnavailableToggleSelector = 'remoteClusterFormSkipUnavailableFormToggle';
+    return {
+      skipUnavailableSwitch: {
+        exists: () => exists(skipUnavailableToggleSelector),
+        toggle: () => {
+          act(() => {
+            form.toggleEuiSwitch(skipUnavailableToggleSelector);
+          });
+          component.update();
+        },
+        isChecked: () => find(skipUnavailableToggleSelector).props()['aria-checked'],
+      },
+    };
+  };
+
+  const createConnectionModeActions = () => {
+    const connectionModeToggleSelector = 'remoteClusterFormConnectionModeToggle';
+    return {
+      connectionModeSwitch: {
+        exists: () => exists(connectionModeToggleSelector),
+        toggle: () => {
+          act(() => {
+            form.toggleEuiSwitch(connectionModeToggleSelector);
+          });
+          component.update();
+        },
+        isChecked: () => find(connectionModeToggleSelector).props()['aria-checked'],
+      },
+    };
+  };
+
+  const createCloudUrlSwitchActions = () => {
+    const cloudUrlSelector = 'remoteClusterFormCloudUrlToggle';
+    return {
+      cloudUrlSwitch: {
+        exists: () => exists(cloudUrlSelector),
+        toggle: () => {
+          act(() => {
+            form.toggleEuiSwitch(cloudUrlSelector);
+          });
+          component.update();
+        },
+        isChecked: () => find(cloudUrlSelector).props()['aria-checked'],
+      },
+    };
+  };
+
+  const createSeedsInputActions = () => {
+    const seedsInputSelector = 'remoteClusterFormSeedsInput';
+    return {
+      seedsInput: {
+        setValue: (seed: string) => form.setComboBoxValue(seedsInputSelector, seed),
+        getValue: () => find(seedsInputSelector).text(),
+      },
+    };
+  };
+
+  const setProxyAddress = (proxyAddress: string) =>
```

Comment:
To follow the same convention as the other element actions, I could also see changing this to:

```
  const createProxyInputActions = () => {
    return {
      proxyInput: {
        setValue: (proxyAddress: string)  => form.setInputValue('remoteClusterFormProxyAddressInput', proxyAddress);
      }
    }
  }

```